jetpack.php

```diff
@@ -221,6 +221,7 @@ function jetpack_admin_missing_autoloader() { ?>
 require_once( JETPACK__PLUGIN_DIR . 'class.jetpack-error.php'         );
 require_once( JETPACK__PLUGIN_DIR . 'class.jetpack-heartbeat.php'     );
 require_once( JETPACK__PLUGIN_DIR . 'class.photon.php'                );
+require_once( JETPACK__PLUGIN_DIR . 'pre-dna-compatibility.php'       );
```

Comment:
maybe `pre-autoload`?

Reply:
why can't most of these classes be autoloaded from a package? There's no reason `Jetpack_Client` can't live in `packages/compat/legacy/class.jetpack-client.php`, and if we need to make the original _file_ available (e.g. because it's being require'd), it can just be an empty `class.jetpack-client.php` in the Jetpack plugin root, with content like:

```php
<?php
// this file has been removed. 
// compat file: packages/compat/legacy/class.jetpack-client.php
// new file: packages/connection/src/Client.php
```